docs/master/api-reference/directives.md

```diff
@@ -1756,6 +1757,16 @@ type Mutation {
 }
 ```
 
+If you have an array you need to inject a value into, you can use an asterisk `*`. 
+
+```graphql
+type Mutation {
+  updateUser(input: UpdateUser!): Task
+    @update
+    @inject(context: "user.id", name: "input.tasks.create.*.user_id")
+}
```

Comment:
Makes sense. Perhaps we should check whether the fields are mandatory, and if not, just `break` instead of throwing an error?

Reply:
I guess the same "issue" could exists with just dot notation. But it would make sense to discard it if an array does not exists at the asterisk segment.

Reply:
With just dot notation, we force the value at the defined position - and create intermediary inputs if necessary. I guess this could not be desired in every case, but it is how it currently works. At least it is well defined.

In the case of an array, we have the problem that it is less clear what to insert. What number of arguments at the desired position should be added? 1 or another number? I would argue it should always be 0 - that being equivalent to injecting nothing at all.